src/NSrtm.Core/Level.cs

```diff
@@ -0,0 +1,9 @@
+namespace NSrtm.Core
+{
+    public enum Level
+    {
+        EllipsoidWgs84,
+        GeoidEgm2008,
+        Terrain
```

Comment:
AFAIK Egm supports multiple formats not only 2008.

Wgs 84 is a ellipsoid with specific radius, so name EllipsoidWgs84 is reduplication. Same with Egm.